app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -689,6 +702,17 @@ struct DeviceDelegateOpenXR::State {
           controllersReadyCallback = nullptr;
       }
   }
+
+  void InitPassthrough() {
+    if (!passthroughSupported || session == nullptr)
+      return;
+
+    assert(OpenXRExtensions::sXrCreatePassthroughFB != nullptr && passthrough == XR_NULL_HANDLE);
+
+    XrPassthroughCreateInfoFB passthroughCreateInfo = {XR_TYPE_PASSTHROUGH_CREATE_INFO_FB};
+    passthroughCreateInfo.flags = XR_PASSTHROUGH_IS_RUNNING_AT_CREATION_BIT_FB;
```

Comment:
You can do this in a single initialization, something like
createinfo = { .type= XR_TYPE..., .flags= XR_PASS...}

and if you do it in the same order as the definition of the struct you can even get rid of the ".type, .flags" etc. Anyway not a big deal, you can leave it as it if you prefer.

Reply:
Ok, I will use the filed names version which should be in general more robust.